docs/index.html

```diff
@@ -108,26 +107,47 @@ <h2 class="cta-heading text-white">UltraStar Format Specification (draft) (offic
                         <div class="image-right-slide-bg clearfix" style="background-image:url(images/mockuo2.png)">
                             <div class="col-md-8">
                                 <h2 class="cta-heading text-white">Let's join forces <br />& shape the future</h2>
-                                <p class="text-white slider-para">US FORMAT is a co-op meta-project for open source
-                                    karaoke scene. It's meant to be for all ultrastar-like karaoke communities.</p>
+                                <p class="text-white slider-para">FORMAT is a co-op project for open source
+                                    karaoke scene. It's <br/>meant to be for all ultrastar-like karaoke communities.</p>
                                 <p class="text-white slider-para">We want everyone to be on board to shape the future of
                                     UltraStar songs.</p>
                                 <div class="cta-btn-group">
                                     <a class="btn btn-white gradient-white border-none btn-rounded-corner"
                                         href="https://github.com/UltraStar-Deluxe/format">GitHub<span
                                             class="icon-on-button"><i class="ion-ios-arrow-thin-right"></i></span></a>
-                                    <a class="btn btn-orange border-none btn-rounded-corner" href="#who">who<span
+                                    <a class="btn btn-orange border-none btn-rounded-corner" href="#together-with">who<span
                                             class="icon-on-button"><i
                                                 class=" glyphicon glyphicon-question-sign"></i></span></a>
-                                    <!--<a class="btn btn-white border-none btn-rounded-corner" href="https://github.com/UltraStar-Deluxe/format">Go To GitHub<span class="icon-on-button"><i class="ion-ios-arrow-thin-right"></i></span></a>-->
                                 </div>
                             </div>
                         </div>
                     </div>
                 </div>
             </div>
-
-
+            <!--
```

Comment:
Please add a comment saying why we need this and when we want to remove it :) (after the `<!-- your comment`

Reply:
done: comment added: this section has to be released when we release format version 1.1.0